cloud-regionsrv-client.spec

```diff
@@ -16,7 +16,7 @@
 #
 
 
-%define base_version 10.0.0
+%define base_version 10.1.0
```

Comment:
There's no change to the base code. Please revert to 10.0.0

Reply:
Updated the wrong version, fixed it now.
Version updated for the plugin-azure